packages/backend/src/apps/notion/dynamic-data/list-database-items/index.ts

```diff
@@ -0,0 +1,46 @@
+import { IGlobalVariable, IJSONObject } from '@automatisch/types';
+
+type Payload = {
+  start_cursor?: string;
+};
+
+export default {
+  name: 'List database items',
+  key: 'listDatabaseItems',
+
+  async run($: IGlobalVariable) {
+    const databases: {
+      data: IJSONObject[];
+      error: IJSONObject | null;
+    } = {
+      data: [],
+      error: null,
+    };
+    const payload: Payload = {
+      start_cursor: undefined as unknown as string,
+    };
+    const databaseId = $.step.parameters.databaseId as string;
+
+    if (!databaseId) {
+      return databases;
+    }
+
+    do {
+      const response = await $.http.post(
+        `/v1/databases/${databaseId}/query`,
+        payload
+      );
+
+      payload.start_cursor = response.data.next_cursor;
+
+      for (const database of response.data.results) {
+        databases.data.push({
+          value: database.id as string,
+          name: database.properties.Name.title[0].plain_text,
```

Comment:
This throws an unexpected error as there may be entries without any "title" set.